src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -847,10 +765,15 @@ def main(argv):
     if os.path.exists(default_dict_path):
       arguments.append(constants.DICT_FLAG + default_dict_path)
 
+  # Strategy pool is the list of strategies that we attempt to enable, whereas
```

Comment:
nit: Using "list" here to describe "Strategy pool" is a little misleading.

Reply:
Still think that we can come up with a better word for this than list because even list might be misleading. Going to keep strategy pool until we have a better way to distinguish chosen and implemented strategies